.github/workflows/release.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Could you add a comment that this is required to fetch tags? Because it's completely non-obvious.

Reply:
While it is a documented behavior of the action, I agree that it is quite unexpected 😅 I added a comment as requested.